appveyor.yml

```diff
@@ -0,0 +1,19 @@
+environment:
+  matrix:
+    - nodejs_version: "0.12"
+    - nodejs_version: "4"
+    - nodejs_version: "6"
+    - nodejs_version: "7"
+
+install:
+  - ps: Install-Product node $env:nodejs_version
+  - npm install
+
+test_script:
+  - node --version
+  - npm --version
+  - npm test
+  - npm run upload
```

Comment:
What happens if someone does a PR with a change to `package.json`? Secure variables (i.e. the GH token) are not decoded for public PRs. Maybe add:

```yaml
branches:
  only:
    - master
```

Or an equivalent in js.

Might also wanna add [`skip_tags: true`](https://www.appveyor.com/docs/branches/#build-on-tags-github-and-gitlab-only).